build.gradle

```diff
@@ -1,12 +1,12 @@
 
 buildscript {
-    ext.kotlin_version = '1.5.0'
+    ext.kotlin_version = '1.6.10'
```

Comment:
I can't build the project, Does it necessary to update kotlin & compileSdkVersion versions?